example/index.editor.jsx

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable */
+const React = require('react');
+const ReactDOM = require('react-dom');
+const SlateEditor = require('./editor').default;
+const Markdown = require('../packages/markdown');
+
+// const DOCBODY = require('./fixtures/markdown');
+const DOCBODY = {
+  magic: require('./fixtures/markdown.magic').default,
+  pure: require('./fixtures/markdown.pure').default,
+  rdmd: {
+    callouts: require('./fixtures/rdmd.callouts').default,
+  }
+};
+
+window.Slate = require('slate'); // @todo remove this
+require('./editor/blocks');
+
+function render(Component = "div", props = {}) {
+  return ReactDOM.render(
+    <Component {...props} />,
+    document.getElementById("root")
+  );
+}
+
+const
```

Comment:
Lets get the tests running on this branch, and make sure this file gets run through eslint/prettier! We always define one variable per line.